images/crane/tests/05-config.sh

```diff
@@ -8,3 +8,6 @@ if [[ "${IMAGE_NAME}" == "" ]]; then
 fi
 
 docker run --rm "${IMAGE_NAME}" config cgr.dev/chainguard/static
+
+echo "This is a failure."
+exit 5
```

Comment:
DO NOT SUBMIT

I have induced this failure for illustrative purposes: 
<img width="475" alt="image" src="https://github.com/chainguard-images/images/assets/2442466/c94440fc-71f2-469f-85e0-3a3890925f45">